Configuration/Eras/python/Era_Run3_cff.py

```diff
@@ -6,6 +6,8 @@
 from Configuration.Eras.Modifier_run3_HB_cff import run3_HB
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 from Configuration.Eras.Modifier_hcalHardcodeConditions_cff import hcalHardcodeConditions
+from Configuration.Eras.Modifier_run2_SiPixel_2018_cff import run2_SiPixel_2018
 
-Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017]), run3_common, run3_GEM, run3_HB, hcalHardcodeConditions)
+
+Run3 = cms.ModifierChain(Run2_2018.copyAndExclude([run2_GEM_2017]), run3_common, run3_GEM, run3_HB, hcalHardcodeConditions, run2_SiPixel_2018)
```

Comment:
The `Run3` era starts from `Run2_2018` already. Therefore, `run2_SiPixel_2018` does not need to be added here, since it is already added to `Run2_2018`. Please revert the change to this file.

Reply:
thanks @kpedro88, fixed by Marco